appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/palette/YoungAndroidPalettePanel.java

```diff
@@ -206,88 +165,7 @@ public void removeComponent(String componentTypeName) {
     removePaletteItem(simplePaletteItems.get(componentTypeName), category);
   }
 
-  /*
```

Comment:
Moved createPropertyEditor(String editorType) to PropertiesUtil.java .

Was necessary for creating property editor from any class and not just in from this class.